### DISCUSSION TITLE
Invitatory Antiphon repetition

### DISCUSSION THREAD

**Discussion by ajmalton** [DivinumOfficium/divinum-officium#2560] [General]
On ordinary Monday (the most recent was 6-7-2021) the invitatory antiphon is Venite, exsultemus Domino.

The psalm then continues Iubilemus Deo salutari nostro, without repetition of the antiphon, which is of course also the first half of the verse.

So far so good: on the site, this is done correctly: but only for the first language.  For the second language (even if it's Latin), the whole first verse is included.

(To reproduce, put the above date on the launch page, then adjust options and look at the beginning of Matins. Put the same language first, then second, and notice the difference.)

1.  This is caused by a programming error discovered during the introduction of use strict.  It is readily fixed.

But ... in all the other languages, Ps.94/95:1a is _not_ textually identical to the antiphon.  Elsewhere in the recitation of the Office (as I learned it) it is only textual _identity_ that causes the repetition of part of a verse to be omitted.  Thus, this bug (for bug it surely is) usually looks "right".  Except when lang2=Latin.

2. Should the suppression of a half-verse repetition, when the antiphon is identical, be varied according to language, or should it follow the Latin?

**Comment by APMarcello3** [comment]
In re: 2, as I inspect the Bea psalter today (19 July 2021) in my printed Maison Mame breviary at Vespers, the final psalm (121) does not match textually the antiphon, and therefore the half-verse is not suppressed. So, it would appear that the suppression should vary according to language.

**Comment by ajmalton** [comment]
Very good: I think that's exactly what we do, everywhere except in this one place at Matins. Unfortunately the mechanism for doing so (viz., the antiphon-et-psalmus procedure) doesn't apply well to the invitatory, so it will have to be a special case.
